promql/bench_test.go

```diff
@@ -32,7 +32,7 @@ func BenchmarkRangeQuery(b *testing.B) {
 	opts := EngineOpts{
 		Logger:     nil,
 		Reg:        nil,
-		MaxSamples: 50000000,
+		MaxSamples: 5000000,
```

Comment:
Why are you changing this?

Reply:
At least two different reasons.

* It helped to complete the test on an EC2 instance with less RAM and before patching it.
* It makes the limit kick-in for the count_over_time query added without adding a lot more 9s.

As the query timeout didn't align with the default I felt it is acceptable to change it. It's also high enough that none of the current tests run into this limit.